src/expandableCalendar/WeekCalendar/index.tsx

```diff
@@ -1,222 +1,239 @@
-import memoize from 'memoize-one';
```

Comment:
Why change the order of imports? We have an order - js libs, react and RN libs, internal services, and tools, internal components, component-specific imports